sources/LocalFiles.ts

```diff
@@ -276,4 +296,30 @@ export default class LocalFiles {
 
 		return res;
 	}
+
+	private getNewName(name: string, counter: number, type: string) : string {
+		// filepath.extname grabs the characters after the last dot (app.css.gz return .gz, not .css.gz)
+		const ext = type === "file" ? name.substring(name.indexOf(".")) : "";
```

Comment:
What if incoming name is "file(1).txt", currently it will be changed to "file(1)(1).txt" which is strange

Reply:
fixed